openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -158,9 +158,9 @@ def refresh_jwt_cookies(request, response, user):
     """
     Resets the JWT related cookies in the response for the given user.
     """
-    if user.is_authenticated and not user.is_anonymous:
```

Comment:
Is this being removed just to avoid redundancy?  Seems like it would be better to just keep this consistent with
 https://github.com/edx/edx-platform/blob/e4f935aab3f1b9621758131520cacc74158ab99d/openedx/core/djangoapps/user_authn/cookies.py#L143 .
Otherwise, maybe add a WARNING comment, but that could easily be missed.  Note: maybe this method will never be called from another view, so maybe it doesn't matter, but it seems like an issue that could crop-up some day.

Reply:
This shouldn't ever be called by another view. If the user is not authenticated, this code will raise an exception so I have no concerns even if someone tries to call it from another view.